var/spack/repos/builtin/packages/py-mvdtool/package.py

```diff
@@ -15,15 +15,29 @@ class PyMvdtool(PythonPackage):
     git      = "https://github.com/BlueBrain/MVDTool.git"
 
     version('develop', branch='master', submodules=True, clean=False)
+    version('2.3.0', tag='v2.3.0', submodules=True, clean=False)
+    version('2.2.1', tag='v2.2.1', submodules=True, clean=False)
     version('2.2.0', tag='v2.2.0', submodules=True, clean=False)
     version('2.1.0', tag='v2.1.0', submodules=True, clean=False)
     version('2.0.0', tag='v2.0.0', submodules=True, clean=False)
 
+    variant('mpi', default=True, description='Build with support for MPI')
```

Comment:
This may break Spykfunc. Could you fix all `py-mvdtool` dependencies to be `py-mvdtool~mpi` unless required otherwise?